spec/services/libanswers_api_service_spec.rb

```diff
@@ -27,33 +27,73 @@
       end
 
       it 'raises a LibanswersApiError' do
-        expect { described_class.new.admin_create_curation_ticket('curation', work.id) }
+        expect { described_class.new.admin_create_curation_ticket(work.id) }
           .to raise_error LibanswersApiService::LibanswersApiError, 'Error Message'
       end
     end
 
-    context 'when a ticket type is passed in' do
+    context 'when called' do
       let!(:mock_faraday_connection) { instance_spy('Faraday::Connection') }
       let!(:dummy_response) { OpenStruct.new(env: OpenStruct.new(status: 200, response_body: '{"ticketUrl": "https://psu.libanswers.com/admin/ticket?qid=13226122"}')) }
-      let!(:accessibility_quid) { '2590' }
       let!(:curation_quid) { '5477' }
 
       before do
         allow(Faraday).to receive(:new).and_return mock_faraday_connection
         allow(mock_faraday_connection).to receive(:post).and_return dummy_response
       end
 
-      it 'uses id 5477 for curation types' do
-        described_class.new.admin_create_curation_ticket('curation', work.id)
+      it 'uses id 5477 for quid and ScholarSphere Deposit Curation for question' do
+        described_class.new.admin_create_curation_ticket(work.id)
         expect(mock_faraday_connection).to have_received(:post).with(
           '/api/1.1/ticket/create',
           "quid=#{curation_quid}&pquestion=ScholarSphere Deposit Curation: #{
           work.latest_version.title}&pname=#{work.display_name}&pemail=#{work.email}"
         )
       end
+    end
+  end
+
+  describe '#admin_create_accessibility_ticket' do
+    let!(:user) { create(:user, access_id: 'test', email: 'test@psu.edu') }
+    let!(:work) { create(:work, depositor: user.actor) }
+    let!(:base_url) { 'www.example.com' }
+
+    context 'when successful response is returned from libanswers /ticket/create endpoint' do
+      it 'returns the libanswers ticket url' do
+        expect(described_class.new.admin_create_accessibility_ticket(work.id, base_url)).to eq 'https://psu.libanswers.com/admin/ticket?qid=15802188'
+      end
+    end
```

Comment:
We should test when there is a report link, too.  I believe this, and the test below, just tests when there isn't a report.